portfolio/src/main/webapp/index.html

```diff
@@ -25,10 +25,9 @@ <h3 id="header"><i>About me</i> &#128515;:</h3>
       <hr>
       <h3 id="header"><i>Fun facts</i> &#128175;:</h3>      
       <p>Click here to get a random <strong>fact</strong> about me: 
-        <button style="background-color: rgba(105, 105, 105, 0.205);" onclick="addRandomFact()">&#128272;</button>
+        <button style="background-color: rgba(105, 105, 105, 0.205);" onclick="getRandomFact();">&#128272;</button>
```

Comment:
Nit: I don't think the semicolon should be necessary here? It doesn't hurt, but we usually add it only if we want to run two functions.